Tasks/codepush-release-cordova/task.json

```diff
@@ -0,0 +1,173 @@
+{
+    "id": "F5990527-F512-4C14-9F8E-1254240DC3CB",
+    "name": "CodePushReleaseCordova",
+    "friendlyName": "CodePush - Release (Cordova)",
+    "description": "Release an update to a CodePush Cordova app deployment",
+    "author": "Microsoft Corporation",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "demands": [],
+    "version": {
+        "Major": "0",
+        "Minor": "0",
+        "Patch": "1"
+    },
+    "minimumAgentVersion": "1.83.0",
+    "instanceNameFormat": "Prepare and release cordova update to $(deploymentName) ($(appName))",
```

Comment:
I would make this the same as the "vanilla" release command. The "(Cordova)" part is already in the task name, so I don't think we need to worry about adding that detail to the instance name.